### ISSUE TITLE
How to flag an outgoing call as "Video"

### ISSUE THREAD

**Issue by ippa** [ianlin/react-native-callkit#46]
First off, deep thanks for a great plugin and your contributions to the react-native community! 

I'm making a videochat application. When my app receives an incoming I call `displayIncomingCall()` with `hasVideo = true`. in Recents I see the incoming call flagged as `Video` which is nice. When registering an outgoing call with `startCall()` I don't have that option though.. and in Recent it shows up as `Audio`. Is there a way to start an outgoing call and flag it as `Video`?

**Comment by ippa** [comment]
ping @ianlin is this plugin still being worked on?

**Comment by zxcpoiu** [comment]
yes, we are keep using this but have no time to develope new features.
this plugin works well without issue in our use case.

**Comment by ippa** [comment]
Thanks for the answer @zxcpoiu , I understand.
